src/EventExport/FileWriter/TabularDataEventFormatter.php

```diff
@@ -56,4 +65,293 @@ public function emptyRow()
+                    return $event->endDate;
+                },
+                'property' => 'endDate'
+            ],
+            'calendarType' => [
+                'name' => 'tijd type',
+                'include' => function ($event) {
+                    return $event->calendarType;
+                },
+                'property' => 'calendarType'
+            ],
+            'location' => [
+                'name' => 'locatie naam',
+                'include' => function ($event) {
+                    if (property_exists($event, 'location') && isset($event->location->name)) {
+                        return $event->location->name;
+                    }
+                },
+                'property' => 'location'
+            ],
+            'address' => [
+                'name' => 'adres',
+                'include' => function ($event) {
+                    if (property_exists($event, 'location') &&
+                        isset($event->location->address)
+                    ) {
+                        $address = [];
+                        if (isset($event->location->address->streetAddress)) {
+                            $address[] = $event->location->address->streetAddress;
+                        }
+
+                        $line2 = [];
+                        if (isset($event->location->address->postalCode)) {
+                            $line2[] = $event->location->address->postalCode;
+                        }
+
+                        if (isset($event->location->address->addressLocality)) {
+                            $line2[] = $event->location->address->addressLocality;
+                        }
+
+                        if (!empty($line2)) {
+                            $address[] = implode(' ', $line2);
+                        }
+
+                        if (isset($event->location->address->addressCountry)) {
+                            $address[] = $event->location->address->addressCountry;
+                        }
+
+                        return implode("\r\n", $address);
+                    }
+                },
+                'property' => 'location'
+            ],
+            'address.streetAddress' => [
+                'name' => 'straat',
+                'include' => function ($event) {
+                    if (isset($event->location->address->streetAddress)) {
+                        return $event->location->address->streetAddress;
+                    }
+                },
+                'property' => 'address.streetAddress'
+            ],
+            'address.postalCode' => [
+                'name' => 'postcode',
+                'include' => function ($event) {
+                    if (isset($event->location->address->postalCode)) {
+                        return $event->location->address->postalCode;
+                    }
+                },
+                'property' => 'address.postalCode'
+            ],
+            'address.addressLocality' => [
```

Comment:
The column header 'regio' seems a bit strange here, not? I'd call it 'gemeente'.